### PR TITLE
ANALYTICS-4695 Add warning

### PR DESCRIPTION
**Pull request by brianroth-rl**
**References**: [ANALYTICS-4695](https://jira.gannett.com/browse/ANALYTICS-4695)

**Code PR**: None

**Description**:
The frequency with which our users create webhooks and expect them to be compatible with external services is alarming. Either our documentation is not clear and we are misleading our users, or they aren't reading our documentation. We have little control over user their behavior, but we can improve the documentation.

warning

### CODE REVIEW

File: source/includes/_edge_webhook.md [hunk 1]
```diff
@@ -27,7 +27,11 @@ Webhook Components:
 - Which lead types should trigger a webhook?
   - Leads can be filtered to only send Calls, only Chats, only Forms & Emails, or any combination of these three options.
 - Payload Format
-  - The format that we will send the data in. This can either be sent in JSON, which is the standard format, or in ADF/XML.
-- The webhook can be set to Active by checking the box next to "Active". If left unchecked, the webhook will not send data.
+  - The [structure of the data in the webhook](#lead-info-apis-webhooks-contacts-lead-apis-response-data-details-nbsp) is the same as that returned by the API. This can either be sent in JSON, which is the standard format, or in ADF/XML.
```
Comment (NBnfork) [inline source/includes/_edge_webhook.md:30]: Where does this go? `[structure of the data in the webhook](#lead-info-apis-webhooks-contacts-lead-apis-response-data-details-nbsp)`

Comment (brianroth-rl) [reply source/includes/_edge_webhook.md:30]: https://doc.api.localiq.com/#lead-info-apis-webhooks-contacts-lead-apis-contact-interactions-leads

Comment (brianroth-rl) [reply source/includes/_edge_webhook.md:30]: I think you want to reconsider if you look at the length of the section I linked to

**Review by NBnfork — APPROVED** [review]
Looks good.

**Pull request merged**